package.json

```diff
@@ -303,7 +303,13 @@
 					"scope": "resource",
 					"default": true,
 					"description": "Automatically highlight tested/untested converage lines with light red/green backgrounds per line for *.lst files."
-				},
+        },
+        "d.coverageReportPath": {
+          "type": "string",
+          "scope": "resource",
+          "default": ".",
+          "description": "Coverage reports path, use then runtime option '--DRT-covopt=\"dstpath:<path>\"' is used."
+        },
```

Comment:
Seems like you accidentally mixed spaces into a tab indented file.

Reply:
oops